.github/workflows/tests.yaml

```diff
@@ -62,6 +58,18 @@ jobs:
 
       - run: poetry check --lock
 
+  build:
+    name: Check build
+    runs-on: ubuntu-latest
+    if: needs.changes.outputs.src == 'true'
```

Comment:
Don't think checking on src changes is valuable. Only thing that really impacts a build to produce bad artifacts is changes to `pyproject.toml` right?

Reply:
I would ask if this is really adding any value for us especially if `poetry check` passes.

Reply:
You're right, this should be `project`.